components/Article/Codebox/index.tsx

```diff
@@ -1,13 +1,18 @@
 import { useEffect, useState } from 'react';
-import { FormattedMessage } from 'react-intl';
 import { highlight, languages } from 'prismjs';
 import classnames from 'classnames';
+import { FaRegCopy, FaCheck } from 'react-icons/fa';
+import { TbCopy, TbCheck } from 'react-icons/tb';
+import { useIntl } from 'react-intl';
 import styles from './index.module.scss';
 import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
 import type { FC, PropsWithChildren, ReactElement, MouseEvent } from 'react';
+import 'prismjs/components/prism-bash';
```

Comment:
Why are we importing this here?

Reply:
Prism.js doesn't come with syntax highlighting for bash or zsh by default, and we need it in `Shellbox`

Reply:
Can you give an example of what you meant by not having syntax highlighting for bash?

Reply:
What bothers me is that we're loading this at bundle time even if we don't need it.

I would if possible, create a dynamic loader (utility file) that loads prismsjs modules on demand.

```tsx
export const loadPrismLanguages = async (language: string) => {
  switch (language) {
    case 'bash':
    case 'zsh':
      await import('prismjs/components/prism-bash');
      break;
    default:
      break;
  }
}
```

Something like that. We can do a call later today on Discord, wdyt?